lnschema_core/_core.py

```diff
@@ -194,7 +194,7 @@ def path(self) -> Union[Path, CloudPath]:
         """Path on storage."""
         return filepath_from_dobject(self)
 
-    def load(self, stream: bool = False):
+    def load(self, stream: bool = False, track_runin: bool = False):
```

Comment:
Could we call this `is_run_input` so that users see the correspondence to `Run.input`?

I think `runin` is a bit cryptic.

Reply:
(It wasn't when the `RunIn` table was still user-facing, but most users will never see it these days.)